pkg/manager/member/pd_member_manager.go

```diff
@@ -588,7 +588,7 @@ func getNewPDSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap) (
 	podAnnotations := CombineAnnotations(controller.AnnProm(2379), basePDSpec.Annotations())
 	stsAnnotations := getStsAnnotations(tc, label.PDLabelVal)
 	storageClassName := tc.Spec.PD.StorageClassName
-	if storageClassName == nil {
+	if storageClassName == nil && len(controller.DefaultStorageClassName) > 0 {
```

Comment:
if `tc.Spec.PD.StorageClassName` is not set, try to use the default storage class of tidb-operator. If the default value of tidb-operator is not set, try to use the default storage class of Kubernetes cluster.